central/processbaseline/service/service_impl.go

```diff
@@ -189,9 +221,29 @@ func (s *serviceImpl) DeleteProcessBaselines(ctx context.Context, request *v1.De
 		return response, nil
 	}
 
-	if err := s.dataStore.RemoveProcessBaselinesByIDs(ctx, search.ResultsToIDs(results)); err != nil {
-		return nil, err
+	// go through list of IDs returned from the search results; clear the baseline and remove deployments from observation.
+	for _, r := range results {
+		key, _ := datastore.IDToKey(r.ID)
+
+		// make sure the deployment still exists, if not that process will take care fo the baseline
+		_, exists, err := s.deployments.GetDeployment(ctx, key.GetDeploymentId())
+
+		if exists && err == nil {
+			// Clear the contents of the baseline
+			_, err := s.dataStore.ClearProcessBaseline(ctx, key, true)
+			if err != nil {
+				log.Errorf("Error clearing process baseline for %q: %v", key, err)
+				continue
+			}
+
+			// Remove the deployment from observation so everything forward is processed as a risk
```

Comment:
No, actually the opposite.  This will take the deployment out of observation such that all incoming indicators will be checked for risks.  The way the observation queue works is that when a deployment is removed from observation, it is removed from the queue but remains in the map backing the queue.  That way we know we have seen this deployment before and as such not start over.  I will look at adding some commentary to ensure that that fact is clear.